javascript/javascript1/week1/goodboy-oldboy/goodboy-oldboy.js

```diff
@@ -0,0 +1,8 @@
+const dogYearOfBirth = 2012;
+const dogYearFuture = 2045;
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+console.log("Your dog will be " + dogYear + " dog years old in " + dogYearFuture + ".");
```

Comment:
You can try to use something called [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals). It does the same job as string literals (what you've used here), besides it looks a bit more cleaner when having multiple variables. This line for example would like in the following way as a template literal:
```
console.log(`Your dog will be ${dogYear} dog years old in ${dogYearFuture}.`);
```